lib/pbench/cli/agent/commands/tools/base.py

```diff
@@ -0,0 +1,27 @@
+from pbench.agent.base import BaseCommand
+
+
+class ToolCommand(BaseCommand):
+    """Common Tool Class"""
+
+    def __init__(self, context):
+        super(ToolCommand, self).__init__(context)
+
+    def remote(self, path):
+        """List all remotes in a given path"""
+        return [p.name for p in path.iterdir() if p.name != "__trigger__"]
```

Comment:
For these unit tests to pass consistently, I think `remote`, `tools`, and `groups` have to return their lists sorted.

Reply:
fixed